cmds/dummy-oauth/main.go

```diff
@@ -46,9 +48,29 @@ func createGetTokenHandler(privateKey *rsa.PrivateKey) http.Handler {
 		if len(scopes) == 1 {
 			scope = scopes[0]
 		}
+
+		var (
+			issuer  string   = ""
+			issuers []string = params["issuer"]
+		)
+		if len(issuers) == 1 {
+			issuer = issuers[0]
+		}
+		var (
```

Comment:
let's put all of the var blocks at the top of this function. it's getting a little long, and there's no early return so no extra cost in creating them